src/imageResponse.js

```diff
@@ -0,0 +1,147 @@
+require("babel-polyfill");
+
+import config from "config";
+import promisify from "promisify-node";
+import log from "./log";
+import dbCache from "./dbCache";
+import image from "./image";
+import aws from "./aws";
+import {futureDate} from "./helper";
+
+const fs = promisify('fs');
+
+const imagePath = (name) => `${config.get('originals_dir')}/${name}`;
+
+// Determine whether the image exists on disk
+// Returns either true of false
+const doesImageExist = async(name) => {
+  try {
+    await fs.access(imagePath(name), fs.R_OK);
```

Comment:
Minor thing: this doesn't test if the image exists, it tests if the image resizer can read the image. If the image resizer simply doesn't have sufficient permissions then it will also return false.

Most likely this is also what you need in the application by the way, it's just that the function name doesn't match up with the functionality.